extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -521,7 +506,9 @@ export class AmpConsent extends AMP.BaseElement {
           'consentInstanceId': this.consentId_,
           'consentStateValue': getConsentStateValue(storedInfo['consentState']),
           'consentString': storedInfo['consentString'],
+          // isDirty will be deprecated
```

Comment:
I'm actually not sure about this. Need @zhouyx 's input later. But let's not assume this is going away for now.